apps/mux/frontend/src/index.test.tsx

```diff
@@ -41,254 +41,258 @@ const SDK_MOCK = {
 const getById = queryByAttribute.bind(null, 'id');
 const getByName = queryByAttribute.bind(null, 'name');
 
-test('throws an error if required installation parameters are not configured', () => {
-  const mockedSdk = {
-    ...SDK_MOCK,
-    parameters: {
-      installation: {
-        muxAccessTokenId: undefined,
-        muxAccessTokenSecret: undefined,
+describe('Mux frontend app', () => {
```

Comment:
This looks like a lot changed but I just added the tests inside a describe block since after converting to vite, since it didn't seem to be able to find the tests.